bitex/exceptions.py

```diff
@@ -17,23 +17,27 @@ class IncompleteCredentialsWarning(UserWarning):
 
 
 class IncompleteCredentialConfigurationWarning(UserWarning):
-    """Raised when a required parameter for authentication isn't found in a config file."""
+    """Raised when a required parameter for authentication isn't found in a
```

Comment:
this is invalid - Docstrings without parameters should be a single line. (PEP257)